openedx/core/djangoapps/content/block_structure/signals.py

```diff
@@ -15,10 +15,6 @@ def _listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable
     Catches the signal that a course has been published in the module
     store and creates/updates the corresponding cache entry.
     """
-    clear_course_from_cache(course_key)
```

Comment:
Leftover change from when both operations were going to be asynchronous, since the update operation also begins with a clear. I'll put it back in.